lib/Logger/LogAppenderSyslog.cpp

```diff
@@ -59,16 +96,13 @@ LogAppenderSyslog::LogAppenderSyslog(std::string const& facility,
   if ('0' <= facility[0] && facility[0] <= '9') {
     value = StringUtils::int32(facility);
   } else {
-    CODE* ptr = reinterpret_cast<CODE*>(facilitynames);
-
-    while (ptr->c_name != nullptr) {
-      if (strcmp(ptr->c_name, facility.c_str()) == 0) {
-        value = ptr->c_val;
-        break;
-      }
+    value = findSyslogFacilityByName(name);
+  }
 
-      ++ptr;
-    }
+  // Since the return value of ::openlog below is not checked or acted
```

Comment:
It is true, the return value of `openlog` is not checked here. Because `openlog` returns `void`.